airflow/docker_image.go

```diff
@@ -39,8 +39,17 @@ func DockerImageInit(image string) *DockerImage {
 	return &DockerImage{imageName: image}
 }
 
-func (d *DockerImage) Build(config airflowTypes.ImageBuildConfig) error {
+func (d *DockerImage) Build(localImage string, config airflowTypes.ImageBuildConfig) error {
 	// Change to location of Dockerfile
+	if localImage != "" {
```

Comment:
I think this logic should not be part of the Docker image build method, it is a business logic around how the image should be built for `astro dev` command, so it might be more accurate for this to remain in the `airflow/docker.go` itself like before, and thus keeping `ImageHandler` interface agnostic of images.

Reply:
If that's the case I'll probably just scrap this PR